scripts/upgrade

```diff
@@ -179,6 +179,23 @@ ynh_script_progression --message="Reloading NGINX web server..." --weight=1
 
 ynh_systemd_action --service_name=nginx --action=reload
 
+#=================================================
+# SEND A README FOR THE ADMIN
+#=================================================
+ynh_script_progression --message="Sending a readme for the admin..." --weight=1
+
+message="Garradin was successfully upgraded :)
+Garradin becomes Paheko. You can now upgrade Garradin with Paheko ! Don't stay with this repository, it will be no more supported.
+Take a Look at the Paheko repository and read the instructions how to migrate your application Garradin to Paheko:
+https://github.com/YunoHost-Apps/paheko_ynh/
+
+This can only be done from the command-line interface - e.g. through SSH. Once you're connected, you simply have to execute the following:
+`sudo yunohost app upgrade garradin -u https://github.com/YunoHost-Apps/paheko_ynh --debug`
+The --debug option will let you see the full output. If you encounter any issue, please report it aand paste the logs.
+Important: After the migration, you'll have to wait a couple of minutes (at most 3 minutes) before you can start using Paheko.
+
+ynh_send_readme_to_admin "$message"
```

Comment:
You would need the experimental helper: 
https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_send_readme_to_admin/ynh_send_readme_to_admin__2

Reply:
Ok, I have added the experimental_helper in `_common.sh`